charts/harbor-operator/values.yaml

```diff
@@ -252,3 +248,20 @@ tolerations: []
 #   }
 # }`
 affinity: {}
+
+redisoperator:
+  # redisoperator.enabled -- Whether to enabled [Redis Operator](https://github.com/spotahome/redis-operator)
+  enabled: false
+  rbac:
+    install: true
+
+minio-operator:
+  # minio-operator.enabled -- Whether to enabled [MinIO Operator](https://github.com/minio/operator)
+  enabled: false
+
+postgres-operator:
+  # postgres-operator.enabled -- Whether to enabled [Postgres operator](https://github.com/zalando/postgres-operator)
+  enabled: false
+
+  configKubernetes:
+    secret_name_template: "{username}.{cluster}.credentials"
```

Comment:
This will allow us to use the same `secret_name_template` as the `postgres-operator`